Added Notice Board

BAAHUBALI7781 opened the pull request.

BAAHUBALI7781 committed:
Added Notice Board

shivam-51 commented on src/views/AllNotice.vue:
```diff
@@ -0,0 +1,144 @@
+    border-color: #ec3169;
+    box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.26);
+}
+.inp{
+    margin:1rem auto;
+    width:90%;
+    max-width:50rem;
+    padding:15px;
+    border-radius:10px;
+    color:black;
+    /* text-align: center; */
+}
```
Why is this CSS required in this file?

BAAHUBALI7781 replied:
I changed input field to text-area but forgot to remove the input css. I will remove that

shivam-51 commented on src/views/AllNotice.vue:
```diff
@@ -0,0 +1,144 @@
+    
+    
+}
+button{
+    font: inherit;
+    cursor: pointer;
+    border: 1px solid #ff0077;
+    background-color: #ff0077;
+    color: white;
+    padding: 0.05rem 1rem;
+    box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.26);
+}
+button:hover{
+    background-color: #ec3169;
+    border-color: #ec3169;
+    box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.26);
```
Why??

shivam-51 commented on src/views/AllNotice.vue:
```diff
@@ -0,0 +1,144 @@
+      <ul v-if="noticearr.length!==0">
+          <li v-for="not in noticearr" :key="not.id">
+              <div style="text-align:right">
+                  <span style="color:blue"><small>By: {{not.user}}</small></span>
+              </div>
+              <div style="text-align:left">
+                  <span><strong>{{not.notices}}</strong></span>
+              </div>
+              <div style="text-align:right">
+                  <span style="color:blue"><small>{{not.timestamp}}</small></span>
```
Do we want the author name and the time to be at two different locations?
Either make it "Posted by foo on bar" at the top or at the bottom.

BAAHUBALI7781 committed:
Update AllNotice.vue

Made the required changes. I have removed the time, only date will be shown when notice will be posted
